src/psyclone/transformations.py

```diff
@@ -2285,46 +2285,57 @@ def apply(self, sched, opencl=True):
         # All we have to do here is set the flag in the Schedule. When this
         # flag is True PSyclone produces OpenCL at code-generation time.
         sched.opencl = opencl
+
+        # This transformation stores some options into the InvokeSchedule
+        sched.set_opencl_options(options)
```

Comment:
Since this call can raise Exceptions I think we should catch them here and re-raise them as a `TransformationError` for consistency with other transformations.

Reply:
Done